Frames.podspec

```diff
@@ -17,14 +17,16 @@ Pod::Spec.new do |s|
   s.exclude_files = "Classes/Exclude"
   s.resources = 'Source/Resources/**/*'
 
-  s.dependency 'PhoneNumberKit', '~> 3.3.0'
-  s.dependency 'Alamofire', '~> 5.4.0'
-  s.dependency 'CheckoutEventLoggerKit', '~> 1.0.2'
+  s.dependency 'PhoneNumberKit', '~> 3.3'
+  s.dependency 'Alamofire', '~> 5.4'
+  s.dependency 'CheckoutEventLoggerKit', '~> 1.0'
 
   s.test_spec do |t|
     t.source_files = 'Tests/**/*.swift'
     t.dependency 'Mockingjay', '~> 3.0.0-alpha.1'
     t.exclude_files = 'Tests/LinuxMain.swift'
     t.resources = 'Tests/Fixtures/*'
   end
+
+  s.pod_target_xcconfig = { 'VALID_ARCHS' => 'arm64 armv7 x86_64' }
```

Comment:
so we need to update readme for this ?

Reply:
We could but this would be a moot point. No 32bit mac is able to run Xcode 9 so this is technically already covered in that